### PR TITLE
TextBlock content is not visible in browsers with the dark theme

### PR DESCRIPTION
**Pull request by Abhishek-Sharma-Msft**
TextBlock defaults the text color as White correctly but StackPanel background needs theme info.

GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/interop/wasm-javascript-3.md [hunk 1]
```diff
@@ -242,6 +242,12 @@ In the constructor of your wrapper control, add the following:
 Background = new SolidColorBrush(Colors.Transparent);
 ```
 
+### TextBlock content is not visible in browsers with the dark theme
+TextBlock defaults the text color as White correctly but StackPanel background needs theme info.
```
Comment (MartinZikmund) [inline doc/articles/interop/wasm-javascript-3.md:246]: I would reword the end as "`StackPanel` background needs to be set correctly."

**Review by MartinZikmund — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@Abhishek-Sharma-Msft please also sign the CLA (a bit above this, yellow button):

image

**Comment by jeromelaban** [comment]
Thanks for your contribution!

**Comment by jeromelaban** [comment]
@mergifyio backport master
